### ISSUE TITLE
Query re efficency spreadsheet - FairRite 5943000601 + 5943001101

### ISSUE THREAD

**Issue by virtual812** [DG1JAN/UniBalun#5]
If I understand correctly you've found some significant efficiency with this combo.

I'm looking to replicate your results, it would seem you had the smaller toroid sitting inside the larger one, is this correct?
Would this also marginally increase power handling? say 20W SSB / 10W FT8?

Have you any advice for size of copper wire?
Sounds in theory that things might get tight inside the 7.9mm ID of the smaller toroid.

Also any thoughts on good cores for 1:1 and 4:1 baluns running ~ 80M to 10M bands?

**Comment by DG1JAN** [comment]
yes, I think the combination of the FairRite 5943000601 + 5943001101 (or Amidon FT82-43 + FT50-43) will bring a slightly improvement in efficiency according my measurements. I would really appreciate if you (or other OMs) could validate this.

Do to the lower core losses and a bit higher mass of the combined toroids, I guess also the power handling will improve. But I haven't done any measurements yet. I only used a 1:49 with this configuration for 20W portable operation in SSB. I didn't recognize any warming (ambient temp 27°C) during this. (Of course SSB was on a very low duty cycle)

As copper wire I was using 0,63mm, as you can se in the picture in the PDF its tight, but just ok with the 3-to-21 winding.

I haven't done much testing for Baluns, yet. From the efficiency point, the core its much less critical IMHO, as only the common mode current is creating flux. For smal dipols (linked Inv-V) I have used Amidon F82-43 (or FairRite 5943000601) and Würth 74270118. But don't expect to much common mode rejection below the 20m Band with the small cores. (for 14 to 30Mhz i measured ~20 - 30 dB, but only 13dB on 40m and 7dB on 80m)

**Comment by virtual812** [comment]
Thanks Jan, will have a go.
Not done a lot of building & testing before, will read up on how to test.
I think i've seen a kind of back to back test done by you and/or DL2MAN.
Weather is beginning to improve here in Australia so hopefully will be outdoors and on air soon.
QRM at home is terrible!

**Comment by virtual812** [comment]
PCBs ordered, Toroids ordered, struggling to find 0.63 Enameled Copper Wire.
I have 1.0mm and 0.8mm on hand.
I can buy 0.56mm and 0.71mm
Looking at capacitors also, the PDF mentions class 1
My RF knowledge is still elementary, but I'm aware capacitor types matter.

I have some of these in 100pF...
https://www.minikits.com.au/Mica-Series

I'm also looking at buying these - Dielectric Characteristic SL...
https://au.element14.com/tdk/cc45sl3ad101jyvna/disc-capacitor-100pf-1kv-radial/dp/3651151

and / or these  - Dielectric Characteristic	C0G / NP0...
https://au.element14.com/tdk/fg28c0g2e101jnt06/cap-100pf-250v-mlcc-radial/dp/3416508

I'll collect up all the parts and read up on how to test.
My understanding is that testing works best with 2 matching units back to back on a VNA.
It's a crude instrument i think, but i have a nanoVNA that i could try.

I'm also looking at building your balun test tool.
At up to 60MHz do resistor types matter?
Looking at metal film, wire wound and ceramic.
I recall reading somewhere that some kinds of resistors add inductance, the wire wound probably and maybe the metal film?
Any thoughts on that?

I'd be happy to test and share my results.

**Comment by DG1JAN** [comment]
I think 0.56mm is a good choice for the combined cores. 

The Mica caps are fine. I usually use TDK  CC45SL3FD101JYVNA (e.g. from DigiKey)

For testing it is important to use low inductance resistors like metal-oxide. In general I would not recommend using my "Balun Test tool" for >30MHz. In general its better to use single resistors with very shorts leads (as short as possible).

**Comment by schornhe** [comment]
To which spreadsheet are you referring. Can you please provide a link?

**Comment by virtual812** [comment]
> To which spreadsheet are you referring? Can you please provide a link?

It's in the repository.
https://github.com/DG1JAN/UniBalun/blob/main/20230620_UnUn_Efficency_Meas.xlsx

**Comment by virtual812** [comment]
Hi Jan, finally getting around to this.
I'm expecting the toroids to arrive any day.
Just had some questions about the testing method.
I have both a nanoVNA and a TinySA, i also have your balun test tool.
I see a rough idea about how to test here.
Is this similar to the method you used?
I'm still learning a few things.
I don't have it with me currently, but i guess there is a mode where the VNA can output a signal and measure loss on it's incoming port.
Any info you could share on how you tested would be appreciated.
